subprojects/scala/src/integTest/groovy/org/gradle/integtests/fixtures/ScalaCoverage.groovy

```diff
@@ -19,9 +19,12 @@ package org.gradle.integtests.fixtures
 import org.gradle.api.JavaVersion
 
 class ScalaCoverage {
-    static final String NEWEST = "2.13.6"
 
-    static final String[] DEFAULT = [NEWEST, "2.10.7", "2.11.12", "2.12.15", "2.13.0", "2.13.1"]
+    static final String[] SCALA_2 = ["2.11.12", "2.12.15", "2.12.17", "2.13.0", "2.13.1", "2.13.9", "2.13.10"]
+    static final String[] SCALA_3 = ["3.1.3", "3.2.0", "3.2.1"]
```

Comment:
```suggestion
    static final String[] SCALA_2 = ["2.11.12", "2.12.17", "2.13.10"]
    static final String[] SCALA_3 = ["3.1.3", "3.2.1"]
```

🤔 I think testing the latest minor from each release is enough.

Reply:
🤔 I think there was a misunderstanding: For Scala, I think 2.11 and 2.12 are very different. Probably the same is true for 3.1 and 3.2. Though within those "major" versions, the latest patch release should be enough to test.

If you still want to go ahead and test only the latest of the major versions, I don't have a strong objection either.

Reply:
I think your suggestion is still valid. The minors in Scala do differ, especially in the way they handle compiler flags, so we need to test at least one of them. Then for each minor testing the latest patch makes sense.

I will also keep the `LATEST_IN_MAJOR` set of versions, because some integration tests do not actually test things related to the particular Scala version, but rather they test the `Scala(Base)Plugin` and the project wiring. However, to make sure those tests do not go stale with time they will depend on just the latest majors, which I think is enough.